tools/bundle_adjust_tracks.cxx

```diff
@@ -180,6 +180,9 @@ static kwiver::vital::config_block_sptr default_config()
                     "when updating cameras. This option is only relevent if a "
                     "value is give to the input_pos_files option.");
 
+
+      config->set_value("krtd_clean_up", "false",
+                        "Delete krtd files present into output_krtd_dir before launching the bundle adjustment.");
```

Comment:
For the description I would say

    Delete all previously existing KRTD files present in output_krtd_dir before writing new KRTD files